pytest-embedded-nuttx/pytest_embedded_nuttx/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""Make pytest-embedded plugin work with NuttX."""
+
+import importlib
+
+from pytest_embedded.utils import lazy_load
+
+from .app import NuttxApp
+
+__getattr__ = lazy_load(
+    importlib.import_module(__name__),
+    {
+        'NuttxApp': NuttxApp,
+    },
+    {
+        'NuttxSerial': '.serial',  # requires 'esp'
+        'NuttxEspDut': '.dut',  # requires 'esp'
+        'NuttxDut': '.dut',  # requires 'serial'
+    },
+)
```

Comment:
`NuttxDut`, and `NuttxEspDut` shall be imported normally,

(although `NuttxEspDut` is activated with service `esp`, but it could be imported normally since it does not require extra dependencies)

`NuttxApp`, and `NuttxSerial` requires esptool / pytest-embedded-serial-esp, so they shall be lazy-loaded.

Reply:
Please check if ok.